packages/shorebird_code_push_protocol/test/src/models/collaborator_test.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:shorebird_code_push_protocol/shorebird_code_push_protocol.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group(Collaborator, () {
+    test('can be (de)serialized', () {
+      const collaborator = Collaborator(
+        userId: 1,
+        email: 'jane.doe@shorebird.dev',
```

Comment:
Does tying this together with email mean users can't change their email in our db?